spec/hashdiff/diff_spec.rb

```diff
@@ -49,6 +49,21 @@
     diff.should == []
   end
 
+  context 'with the ignore_keys option' do
+    a = { a: 1, b: { d: 2, a: 3 }, c: 4 }
+    b = { a: 2, b: { d: 2, a: 7 }, c: 5 }
+
+    it 'ignores a single key' do
+      diff = described_class.diff(a, b, ignore_keys: :a)
+      diff.should == [['~', 'c', 4, 5]]
+    end
```

Comment:
To specify which keys to ignore, it'll be good to have the choice to specify with precision. Could the ignored keys be an array of paths? That seems to complicate the specification/interface.

 Will custom comparison method address the use case? https://github.com/liufengyun/hashdiff?tab=readme-ov-file#specifying-a-custom-comparison-method

Reply:
As I said in #86, a custom comparison can do this, the PR is to add a convenience method. It was my original intention to ignore only keys at the end of a path, but that is more complex, could that be a further enhancement?

An array of paths may need to deal with the :delimiter option and non-existent paths so I tried to keep it simple to start with.

Reply:
As it's useful, I think it's OK to have this simple utility option. We can avoid misunderstanding by making its intended effect more explicit in the documentation and invite users to use custom comparison for more complex usage.